test/tests/rebase.js

```diff
@@ -1891,13 +1891,13 @@ describe("Rebase", function() {
         .then(function(commit) {
           // verify that the "ours" branch has moved to the correct place
           assert.equal(commit.id().toString(),
-          "24250fe6bd8a782ec1aaca8b2c9a2456a90517ed");
+          "617cd03370dd799f372e9dcfcd0b097aede1bd7f");
```

Comment:
I'm not actually sure... I didn't actually touch anything related to it or modify any of the commits and the new shas are consistently. even passing on CI. I could take another look if it's significant.

Reply:
Yes, I would take a look. This means that something changed with how libgit2 creates a commit, since the sha is generated from the content + header.

Reply:
Looks like these commits changed because `createCommitWithSignature` and `rebase.commit` now have the same newline handling. Verified that commits created before this PR, with this PR, and via the CLI are create valid signatures, and that NodeGit before and with this PR and the CLI are able to successfully verify the signatures.